package.json

```diff
@@ -79,6 +79,7 @@
     "boom": "5.2.0",
     "bs58check": "^2.0.2",
     "bson": "1.0.5",
+    "btoa": "^1.2.1",
```

Comment:
since this function is just a one-liner, i'd prefer we just inline it instead of adding a new dependency: `Buffer.from(str.toString(), 'binary').toString('base64')`

Reply:
@maikelmclauflin can you change this?

Reply:
sorry. didn't see this. done @diracdeltas